client/app/components/Resizable/index.jsx

```diff
@@ -53,6 +53,8 @@ export default function Resizable({ toggleShortcut, direction, sizeAttribute, ch
     () => (
       <span
         className={`react-resizable-handle react-resizable-handle-${direction}`}
+        role="gridcell"
```

Comment:
What's the motivation of using `gricell` on this `span`? Maybe role `separator` would be better? Resize handles are tricky.